src/templates/tr/base.html

```diff
@@ -0,0 +1,184 @@
+{% extends "base/base.html" %}
+
+{% block description %}Web Almanak, web topluluğunun uzmanlığını HTTP Archieve verileri ve eğilimleri ile birleştiren yıllık bir web raporudur.{% endblock %}
```

Comment:
We normally say to keep brand names like Web Almanac and HTTP Archive in their original English, but if you think these are better translated then I'm OK with that. What do you think?

Reply:
I think. it is better to keep the brand names as it is.